app/scenes/root/root_container.js

```diff
@@ -16,13 +17,17 @@ function mapStateToProps(state, ownProps) {
         ...ownProps,
         credentials: state.entities.general.credentials,
         logoutRequest: state.requests.users.logout,
-        loginRequest: state.requests.users.login
+        loginRequest: state.requests.users.login,
+        currentTeamId: state.entities.teams.currentId,
+        teams: state.entities.teams.teams
     };
 }
 
 function mapDispatchToProps(dispatch) {
     return {
         actions: bindActionCreators({
+            ...teamActions,
```

Comment:
do we need all the **teamActions**? cause actually I don't see any of them being used in the `root` component